dashboard/app/views/home/_privacy.md

```diff
@@ -275,8 +272,8 @@ If you choose to use our Website from the European Union, please note the follow
 
 ### Conditions of Use and Notices
 
-If you choose to visit our Website, your visit and any dispute over privacy is subject to this Privacy Policy and our Terms of Service, including limitations on damages, arbitration of disputes, application of the law of the State of Washington and venue for all disputes in Seattle, Washington.
-
+If you choose to visit our Website, your visit and any dispute over privacy is subject to this Privacy Policy and our Terms of Service, including limitations on damages, arbitration of disputes, application of the law of the State of Washington and venue for all disputes in Seattle, Washington.  Where we have entered into separate agreements with applicable local education authorities, such as our [Student Data Privacy Addendum](/dpa), the terms of those agreements may also govern and/or take precedence. 
```

Comment:
If this is shown on dashboard, do we need the `/dpa` URL to also include the `https://code.org` domain?

Reply:
ooh good catch